client/fs_endpoint.go

```diff
@@ -447,6 +441,15 @@ OUTER:
 			break OUTER
 		case frame, ok := <-frames:
 			if !ok {
+				// framer may have been closed when an error
```

Comment:
Why is this not needed on the stream method above?

Reply:
Oh! It probably is and my testing just didn't trigger it there! Fixing.